### DISCUSSION TITLE
generate animated svg for demos

### DISCUSSION THREAD

**Discussion by mgaitan** [Textualize/textual#2622] [Ideas]
I know that textual can generate beautiful screenshots of the current state of the application in SVG format. 

I wonder if this functionality could be extended to generate animated SVGs and get "sceencast" for demo purposes embeddable in READMEs.

**Comment by willmcgugan** [comment]
Nothing planned. TBH, I think a video is probably a better target than an animated SVG.

**Reply by jgarte** [reply]
Hi, why do you think a video is a better target?

Just curious to know your thoughts on that.
